requirements.txt

```diff
@@ -1,2 +1,2 @@
-fhirclient
+fhirclient==4.0.0
```

Comment:
If using `STU3` as the default, fhirclient version should be `3.0.0` then (see https://github.com/smart-on-fhir/client-py)

Reply:
Good point. I don't know why it's on 4.0.0. Since 4.0.0 it kind of new it makes sense that the latest version hasn't been published to pypi yet I think.

Reply:
`4.0.0` has been released two years ago on github...

Reply:
you need to change `fhirclient==4.0.0` to `fhirclient==3.0.0` to use the `STU3` as default version

Reply:
changed it. I was assuming everything 3.x.x would be for STU3 but it doesn't say so in the documentation I see now.